packages/one-app-bundler/__tests__/webpack/app/webpack.client.spec.js

```diff
@@ -119,4 +123,15 @@ describe('webpack/app', () => {
     // stringified, also can't use .toHaveProperty() as the key we need has a dot in it
     expect(browserDefinitions[0].definitions['global.BROWSER']).toBe('true');
   });
+
+  it('should use core-js modules only for modern and not generate the legacy build', () => {
+    process.env.NODE_ENV = 'development';
+    getConfigOptions.mockReturnValueOnce({ disableDevelopmentLegacyBundle: true });
+    const modernWebpackConfig = configGenerator('modern').entry.vendors.length;
+
+    getConfigOptions.mockReturnValueOnce({ disableDevelopmentLegacyBundle: true });
+    const legacyWebpackConfig = configGenerator('legacy').entry.vendors.length;
+    expect(legacyWebpackConfig - modernWebpackConfig).toBeGreaterThan(2);
+    expect(legacyWebpackConfig - modernWebpackConfig).toEqual(3);
```

Comment:
I think this might be too flakey, can you be more explicit over what should and should not be there